### PR TITLE
Bugherd 130 132

### PR DESCRIPTION
**Pull request by felixsaintgenis**
The arrow doesn't rotate when opening
https://www.bugherd.com/projects/158157/tasks/130

I think the arrow should be horizontal
https://www.bugherd.com/projects/158157/tasks/132

### CODE REVIEW

File: assembl/static2/js/app/components/cookies/cookiesSelector.jsx [hunk 1]
```diff
@@ -50,7 +50,12 @@ const CookiesSelector = ({
                     handleCategorySelection(category);
                   }}
                 >
-                  <span className={classnames('assembl-icon-right-dir', { 'active-arrow': isActiveKey })} />
+                  <span
+                    className={classNames({
+                      'assembl-icon-down-dir': isActiveKey || show,
+                      'assembl-icon-right-dir': !isActiveKey || !show
+                    })}
+                  />
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/cookies/cookiesSelector.jsx]: Can't we use the same assembl-icon-angle-down and assembl-icon-angle-right here to have the same arrows like in the navigation?

**Comment by vincentfretin** [comment]
Ah like I said in the BH ticket, you should also modify js/app/components/synthesis/sideMenuTree.jsx used for the content tree in the synthesis.

File: assembl/static2/css/views/debate.scss [hunk 2]
```diff
@@ -1,3 +1,17 @@
+.proposals {
+  .assembl-icon-angle-right {
+    &:before {
+      font-size: 30px;
```
Comment (MilanBarande) [inline assembl/static2/css/views/debate.scss]:
```suggestion
# font-size: $font-size-l;
```

File: assembl/static2/css/views/debate.scss [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+    }
+  }
+
+  .assembl-icon-angle-down {
+    &:before {
+      font-size: 30px;
```
Comment (MilanBarande) [inline assembl/static2/css/views/debate.scss]:
```suggestion
      font-size: $font-size-l;
```

File: assembl/static2/js/app/components/cookies/cookiesSelector.jsx (hunk 1 above)
Comment (felixsaintgenis) [reply assembl/static2/js/app/components/cookies/cookiesSelector.jsx]: sure

**Pull request merged**